packages/lesswrong/components/review/FrontpageReviewWidget.tsx

```diff
@@ -270,101 +270,103 @@ const FrontpageReviewWidget = ({classes, showFrontpageItems=true}: {classes: Cla
+              <SettingsButton showIcon={false} label={`How does the ${REVIEW_NAME_IN_SITU} work?`}/>
+            </Link>
           </LWTooltip>
+        </SectionTitle>
+        <div className={classes.reviewTimeline}>
+          <div className={classes.nominationBlock}>
+            <LWTooltip placement="bottom-start" title={nominationsTooltip} className={classNames(classes.progress, {[classes.activeProgress]: activeRange === "NOMINATIONS"})}>
+              <div className={classNames(classes.blockText, classes.blockLabel)}>Preliminary Voting</div>
+              <div className={classNames(classes.blockText, classes.hideOnMobile)}>{nominationEndDate.format('MMM Do')}</div>
+              {activeRange === "NOMINATIONS" && <div
+                className={classes.coloredProgress}
+                style={{width: `${dateFraction(currentDate, nominationStartDate, nominationEndDate)}%`}}
+              />}
+            </LWTooltip>
+          </div>
+          <div className={classes.reviewBlock}>     
+            <LWTooltip placement="bottom-start" title={reviewTooltip} className={classNames(classes.progress, {[classes.activeProgress]: activeRange === "REVIEWS"})}>
+              <div className={classNames(classes.blockText, classes.blockLabel)}>Reviews</div>
+              <div className={classNames(classes.blockText, classes.hideOnMobile)}>{reviewEndDate.format('MMM Do')}</div>
+              {activeRange === "REVIEWS" && <div className={classes.coloredProgress} style={{width: `${dateFraction(currentDate, nominationEndDate, reviewEndDate)}%`}}/>}
+            </LWTooltip>   
+          </div>
+          <div className={classes.votingBlock}>
+            <LWTooltip placement="bottom-start" title={voteTooltip} className={classNames(classes.progress, {[classes.activeProgress]: activeRange === "VOTING"})}>
+              <div className={classNames(classes.blockText, classes.blockLabel)}>Final Voting</div>
+              <div className={classNames(classes.blockText, classes.hideOnMobile)}>{voteEndDate.format('MMM Do')}</div>
+              {activeRange === "VOTING" && <div className={classes.coloredProgress} style={{width: `${dateFraction(currentDate, reviewEndDate, voteEndDate)}%`}}/>}
+            </LWTooltip>
+          </div>
         </div>
-      </div>
-      
-      {/* Post list */}
-      {showFrontpageItems && <AnalyticsContext listContext={`LessWrong ${REVIEW_YEAR} Review`} capturePostItemOnMount>
-        {/* TODO:(Review) I think we can improve this */}
-        <RecommendationsList algorithm={getReviewAlgorithm()} />
-      </AnalyticsContext>}
-
-      {/* TODO: Improve logged out user experience */}
-      
-      {activeRange === "NOMINATIONS" && eligibleToNominate(currentUser) && <div className={classes.actionButtonRow}>
         
-        {allPhaseButtons}
+        {/* Post list */}
+        {showFrontpageItems && <AnalyticsContext listContext={`LessWrong ${REVIEW_YEAR} Review`} capturePostItemOnMount>
```

Comment:
Convention is that all Context variables use camelCase, no spaces, e.g. lessWrong${REVIEW_YEAR}Review. However, I think in this case you should separate out year into a separate variable, e.g.
`listContext="lesswrongReview" reviewYear=`${REVIEW_YEAR}``